src/containers/Cech.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useRouteData } from 'react-static';
+import { Link } from '@reach/router';
+
+const CECH = () => {
+  const { cech } = useRouteData();
```

Comment:
```suggestion
  const { project } = useRouteData();
```
I would rename this route data to project so it is descriptive of what the data contains